### ISSUE TITLE
Color customization instructions

### ISSUE THREAD

**Issue by Digharatta** [Platoniq/decidim-install#12]
Thank you for your wonderful work.

May I suggest adding look and feel customization instructions, as mentioned at:

https://github.com/decidim/decidim/issues/3886

**Comment by deivid-rodriguez** [comment]
@Digharatta In reality what I meant by https://github.com/decidim/decidim/issues/3886#issuecomment-406894564, is that it'd be nice for decidim-install to suggest a deployment configuration for the installed app, for example through https://github.com/capistrano/capistrano.

The color customization stuff is already [documented](https://github.com/decidim/decidim/blob/84c7190bb953a420c4d9cc79d0d1e6039e9ce94d/docs/customization/styles.md) in decidim.

**Comment by Digharatta** [comment]
@deivid-rodriguez Well, for me it was challenging to guess the proper filename and to apply the changes.

Capistrano would indeed be fantastic, - for multiple deployments as well.
Something along the lines of https://gorails.com/deploy/ubuntu/18.04#capistrano

**Comment by deivid-rodriguez** [comment]
But the docs in decidim should be clear enough, right? Maybe we can link them.

In my opinion, in the short-mid term, this awesome guide should be merged to the decidim project for discoverability.

**Comment by deivid-rodriguez** [comment]
Yeah, we can improve the generators so some stuff is done for us. But this stuff should be optional, since not everybody will use it. For example, I don't think any of these gems is used by Codegram people.

**Comment by microstudi** [comment]
Maybe we should start to create chapters in this guide, a mini howto's (following the basic-config). I'll be doing that at any time I'll learn something for sure, I haven't tried anything related to customization yet, but I'll be there eventually.
If anyone wants to contribute, you can do it by just point me resource you've found, things you've discovered (like the colour customization) or even start a new document in a pull request. I wan't merge anything until I've tried myself though.
bottom line, we will improve this guide!
Next things I'd like to write about would be (not necessarily in that order):
- capistrano deployment
- maintaining your own Decidim install through git
- changing default configuration after install (like the locales)
- upgrading to new versions
- others?

About the generators, I understand why they don't create the production defaults, all these gems are optionals, are used in this guide because they are (in my opinion) the easiest way to go to production, but for larger instances or other type of installs (heroku) they may change substantially.
